rpm/chum.spec

```diff
@@ -1,4 +1,4 @@
-Summary:        SailfishOS:Chum community repository
+Summary:        Proper SSU configuration for the SailfishOS:Chum community repository
```

Comment:
Don't think we need "Proper" in Summary. As if there are packages without "proper" SSU config.

Reply:
Absolutely agreed: Though I can easily imagine an "improper" SSU configuration (for a fixed SFOS release version or in the "Global" section), without some background knowledge this is impossible to understand.